<ui version="4.0" >
 <class>RenderWidget_UI</class>
 <widget class="QDialog" name="RenderWidget_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Rendering</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Output file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="KUrlRequester" name="out_file" />
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QCheckBox" name="zone_only" >
     <property name="text" >
      <string>Selected zone only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QCheckBox" name="play_after" >
     <property name="text" >
      <string>Play after render</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QCheckBox" name="enable_params" >
       <property name="text" >
        <string>Extra parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="av_params" />
     </item>
    </layout>
   </item>
   <item row="5" column="0" >
    <widget class="KPushButton" name="buttonStart" >
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>153</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="KTabWidget" name="ktabwidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Templates</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="KListWidget" name="format_list" />
       </item>
       <item row="1" column="1" colspan="4" >
        <widget class="KListWidget" name="size_list" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="KLineEdit" name="advanced_params" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QToolButton" name="buttonSave" >
         <property name="text" >
          <string>S</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QToolButton" name="buttonEdit" >
         <property name="text" >
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" >
        <widget class="QToolButton" name="buttonDelete" >
         <property name="text" >
          <string>D</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="tc_overlay" >
     <property name="text" >
      <string>Timecode overlay</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RenderWidget_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RenderWidget_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
